webrtc/videoroom/assets/src/room.ts

```diff
@@ -2,12 +2,14 @@ import "../css/app.scss";
 
 import { MEDIA_CONSTRAINTS, SCREENSHARING_CONSTRAINTS } from "./consts";
 import {
+  addAudioElement,
```

Comment:
It's seems that this line survived the rebase. This PR shouldn't include any changes in `.ts` files.